artman/config/loader.py

```diff
@@ -95,43 +96,38 @@ def _validate_artifact_config(artifact_config):
     pass
 
 
-def _normalize_artifact_config(artifact_config, input_dir):
+def _normalize_artifact_config(artifact_config, artman_config_dir):
     """Normalize the config protobuf based on flags passed from command line.
 
     Note: we are not normalizing output folders because they are no longer
     configurable, and the output folder calculation now happens in converter.
     Once the individual GAPIC output folder becomes configurable, that folder
     name calculation logic should be moved from converter into this method.
     """
-    # Normalize the input file or folder by prefixing with input_dir if needed.
+    # Normalize the input file or folder by concanating relative path with
+    # the folder of artman config yaml.
     artifact_config.service_yaml = _normalize_path(
-        artifact_config.service_yaml, input_dir)
+        artifact_config.service_yaml, artman_config_dir)
 
     artifact_config.gapic_yaml = _normalize_path(
-        artifact_config.gapic_yaml, input_dir)
-
-    if not artifact_config.import_proto_path:
-        artifact_config.import_proto_path[:] = [input_dir]
-    else:
-        normalized_import_proto_path = []
-        for import_proto_path in artifact_config.import_proto_path:
-            normalized_import_proto_path.append(
-                _normalize_path(import_proto_path, input_dir))
-
-        artifact_config.import_proto_path[:] = normalized_import_proto_path
+        artifact_config.gapic_yaml, artman_config_dir)
 
     normalized_src_proto_paths = []
     for src_proto_path in artifact_config.src_proto_paths:
         if src_proto_path.startswith('-'):
             # Retain the exclusion mark "-" as the prefix of the normalized path
             normalized_src_proto_paths.append(
-                '-%s' % _normalize_path(src_proto_path[1:], input_dir))
+                '-%s' % _normalize_path(src_proto_path[1:], artman_config_dir))
         else:
             normalized_src_proto_paths.append(
-                _normalize_path(src_proto_path, input_dir))
+                _normalize_path(src_proto_path, artman_config_dir))
     artifact_config.src_proto_paths[:] = normalized_src_proto_paths
 
     return artifact_config
 
 def _normalize_path(path, prefix):
+  if '..' in path:
+      raise ValueError('Cannot use ".." path-typed field in artman yaml. '
+                       'Please use either path relative to artman yaml '
+                       '(preferred). or an absolute path.')
   return path if os.path.isabs(path) else os.path.join(prefix, path)
```

Comment:
Are we 100% sure that `prefix` was previously passed through `os.path.realpath`?